sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/ScriptTransformationSuite.scala

```diff
@@ -49,6 +52,23 @@ class ScriptTransformationSuite extends SparkPlanTest with TestHiveSingleton {
     outputSerdeClass = Some(classOf[LazySimpleSerDe].getCanonicalName)
   )
 
+  private val uncaughtExceptionHandler = new TestUncaughtExceptionHandler
+
+  protected override def beforeAll(): Unit = {
+    super.beforeAll()
+    Thread.setDefaultUncaughtExceptionHandler(uncaughtExceptionHandler)
```

Comment:
Shall we better get default handler and restore it?

Reply:
ok~ `getDefaultUncaughtExceptionHandler` and store it at `beforeAll` method, then restore use `defaultUncaughtExceptionHandler` at `afterAll` method